sway/tree/view.c

```diff
@@ -583,6 +584,7 @@ void view_unmap(struct sway_view *view) {
 	}
 
 	transaction_commit_dirty();
+	cursor_send_pointer_motion(config->handler_context.seat->cursor, 0, true);
```

Comment:
Can you move this below `view->surface = NULL` so we don't have bad pointers when calling `cursor_send_pointer_motion`?

Reply:
done